ts/DataGrid/DataGrid.ts

```diff
@@ -65,17 +66,57 @@ class DataGrid {
      * @param options
      * The options of the data grid.
      *
-     * @return The new data grid.
+     * @param async
+     * Whether to initialize the dashboard asynchronously. When true, the
+     * function returns a promise that resolves with the dashboard instance.
```

Comment:
This description is strange `Whether to initialize the dashboard asynchronously. When true, the function...`